moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -717,5 +813,8 @@ class LockedPlanningSceneRW : public LockedPlanningSceneRO
   {
     return planning_scene_monitor_->getPlanningScene();
   }
+
+private:
+  std::shared_ptr<UniqueTimedLock> ulock_;
```

Comment:
Open for debate: The shared pointers here copy the old behavior. I'm thinking of not allowing this since it doesn't make a lot of sense to share write access between different entities. I'd suggest making LockedPlanningSceneRW non-copyable. LockedPlanningSceneRO should probably also not be using a shared pointer, acquiring another shared lock seems way more reasonable to me.